data_processors/lims/lambdas/google_lims.py

```diff
@@ -45,12 +48,34 @@ def scheduled_update_handler(event, context) -> Dict[str, int]:
     logger.info("Start processing LIMS update event")
     logger.info(libjson.dumps(event))
 
+    sheets = event.get('sheets', ["Sheet1", ])
+    is_truncate = event.get('truncate', False)
+
+    if not isinstance(sheets, list):
+        _halt(f"Payload error. Must be array of string for sheets. Found: {type(sheets)}")
+
+    if not isinstance(is_truncate, bool):
+        _halt(f"Payload error. Must be boolean for truncate. Found: {type(is_truncate)}")
+
+    truncated = False
```

Comment:
If I see this right, then a request may return an error `Payload error. Must be boolean for truncate..` if a `truncate` option is given and the type is not boolean. This suggests that this is a valid option for a request.
However, if the type issue is resolved and a `True` value is given, it's just ignored/overwritten without feedback?
Why support the `truncate` option in the first place?

Reply:
Removing...